src/kits/datetimeStd/native/std/datetimeStd_DateTimeServiceStd.c

```diff
@@ -41,13 +43,28 @@ int64_t datetimeStd_DateTimeServiceStd_doNow(SedonaVM* vm, Cell* params)
   return nanos;
 }
 
- //  Setting system time not implemented on Win32
-  
 // void doSetClock 
 Cell datetimeStd_DateTimeServiceStd_doSetClock(SedonaVM* vm, Cell* params)
 {
   int64_t nanos = *(int64_t*)(params+0); // param 0+1
-    
+  nanos += SEDONA_EPOCH_OFFSET_SECS * 1000 * 1000 * 1000;
```

Comment:
if possible, I prefer the way of sys kit's native: std folder only keep the generic codes, for platform specific codes, wrap it into a function and implement it in platform specific files. That can make codes more clear.

Reply:
In general I agree with that approach. It makes a lot of sense if native kit implementation is almost completely different on different platforms like unix and win32 (for example, serial). Than different kits are using different source files. No #ifdefs.
But for this simple case I don't see much gain here. I certainly can wrap 10 lines of code (80% code) of datetimeStd_DateTimeServiceStd_doSetClock() into separate function and place it into separate file, but #ifdef _WIN32 will be kept in std implementation. IMHO not nice. 
Another approach would be to copy complete datetimeStd_DateTimeServiceStd implementation to different folder and remove all ifdefs, but then we have several copies of similar code in different places. No easy to maintain at all.

The line you commented on will be in std implementation anyway, because SEDONA_EPOCH_OFFSET_SEC is already used in 
int64_t datetimeStd_DateTimeServiceStd_doNow(SedonaVM* vm, Cell* params)

What do you think?

Reply:
I've moved whole doSetClock native implementation to platform specific file and created default implementation that is used on Win32 platform.

Reply:
Looks better now, thanks!